src/main/resources/application.yml

```diff
@@ -1,6 +1,7 @@
 settings:
   api-key: live_tF1DP6ary6TwsK1YceaeoMQ7B6cRuZFLQr23GFU3ZtVe5AwM8yZSYTkczB4WYXZ2
-
+  folder:
+    root: C:\Users\udmyt\OneDrive\Изображения\Cat Api
```

Comment:
Images are saved, but folder 'Cat Api' was turned into file and singlecan not be opened